tests/parallel_for/hierarchical_new.cpp

```diff
@@ -70,5 +70,9 @@ my_queue.submit([&](handler &cgh) {
   });
 });
 //////// End slide
-    return 0;
+  /* We must wait for for the queue to finish as none of buffer's destruction
+   * are blocking.
```

Comment:
since they do not have anything to write back.
s/are blocking/is blocking/
Remove *